ExceptionCar.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include <iostream>
+#include <exception>
+
+
+class MyException : public std::exception
```

Comment:
Nazwa klasy MyException trochę mało mówi co to za konkretny wyjątek. 
Nazwa pliku ExceptionCar też nie zgadza się z nazwą klasy :)